containerized-build.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-CRUNTIME=${CRUNTIME:-docker}
+CRUNTIME=${CRUNTIME:-"DOCKER_BUILDKIT=1 docker"}
```

Comment:
I would rather put

```
export DOCKER_BUILDKIT=1
```

somewhere else in a separate line. Maybe as the last one before `function do_gen()`.

CI seems to be unhappy about your current solution, but my suggestion should fix it:

https://github.com/rancher-sandbox/lockc/pull/40/checks?check_run_id=3338517679

Reply:
fixed with `env`